Sources/PointFree/Session.swift

```diff
@@ -14,7 +14,10 @@ public func writeSessionCookieMiddleware<A>(_ update: @escaping (Session) -> Ses
       return setCookie(
           key: pointFreeUserSession,
           value: value,
-          options: [.path("/")]
+          options: [
+            .expires(AppEnvironment.current.date().addingTimeInterval(60 * 60 * 24 * 365 * 10)),
```

Comment:
would we even do `Date.distantFuture`?

Reply:
Ha, that's the year 4,000. A little optimistic, are we?

My only worry is some browsers handling this stuff weird and breaking cookies for some folks.